fakeredis/aioredis.py

```diff
@@ -1,5 +1,15 @@
-try:
-    import aioredis
+import redis
+import packaging.version
+
+# aioredis was integrated into redis in version 4.2.0 as redis.asyncio
+if packaging.version.Version(redis.__version__) >= packaging.version.Version("4.2.0"):
+    import redis.asyncio as aioredis
+    from ._aioredis2 import FakeConnection, FakeRedis  # noqa: F401
+else:
+    try:
+        import aioredis
+    except ImportError as e:
+        raise ImportError("aioredis is required for redis-py below 4.2.0") from e
```

Comment:
I wasn't sure what would be a good error to raise here, it might be more valid to have a `RuntimeError` instead of an `ImportError`, keen to hear your thoughts

Reply:
I think this is fine